### DISCUSSION TITLE
Multiple box_dividers in box_plot

### DISCUSSION THREAD

**Discussion by jhexner** [endangeredoxen/fivecentplots#70] [Q&A]
Hi,

In JMP multiple box_dividers are created, not only for the first group. This makes the groups much clearer.
is it possible to achieve similar behavior here?

![jmp_example](https://github.com/endangeredoxen/fivecentplots/assets/101968535/7bb31e2d-ec5c-421e-bd7c-79667ad223ba)

**Comment by endangeredoxen** [comment]
There was some reason I didn't replicate JMP exactly, but I don't recall what it was now.  I'll take a look into it
